driver/src/orderbook/streamed/mod.rs

```diff
@@ -0,0 +1,22 @@
+#![allow(dead_code)]
+
+mod state;
+
+use ethcontract::Address;
+
+type UserId = Address;
+type TokenAddress = Address;
+type OrderId = u16;
+type TokenId = u16;
+type BatchId = u32;
+
+#[cfg(test)]
+mod tests {
+    use serde::{Deserialize, Serialize};
+
+    #[derive(Debug, Deserialize, Serialize)]
+    pub struct EventBackup {
```

Comment:
how is this used?

Reply:
I didn't mean to include this. I was using it to test locally.